src/lib/Menu/ContentEditRightSidebarBuilder.php

```diff
@@ -97,8 +98,9 @@ public function createStructure(array $options): ItemInterface
         /** @var Location $parentLocation */
         $parentLocation = $options['parent_location'];
 
-        $canPublish = $this->permissionResolver->canUser('content', 'publish', $content);
-        $canEdit = $this->permissionResolver->canUser('content', 'edit', $content);
+        $version = (new Version())->translateToAnyLanguageOf([$language->languageCode]);
```

Comment:
Let's not forget that this is a target (I get what happens, but I doubt anyone beside us will ;) ):
```suggestion
        $target = (new Target\Version())->translateToAnyLanguageOf([$language->languageCode]);
```